TrackletAlgorithm/FullMatchMemory.h

```diff
@@ -246,6 +246,29 @@ class FullMatch : public FullMatchBase<FMType>
     data_.range(kFMZResMSB,kFMZResLSB) = zres;
   }
 
```

Comment:
Can you make brace style consistent with clang-format and also unify it in these two functions.